### DISCUSSION TITLE
Are there plans for adding model switcher to Github Copilot Mobile?

### DISCUSSION THREAD

**Discussion by zavocc** [community/community#143242] [Copilot Conversations]
Now, I have access to O1 models and Claude 3.5 as per https://github.com/newsroom/press-releases/github-universe-2024 

Right now I can only access these models in Copilot in GitHub.com/Copilot and in VSC. Are there plans to add this to GitHub mobile as well?

**Comment by SyedAliJunaidSherazi** [comment]
Hi Zavocc,

That’s exciting news that you now have access to O1 models and Claude 3.5! Currently, these models are available through GitHub Copilot on GitHub.com/Copilot and within Visual Studio Code. As for plans to expand access to GitHub mobile, there hasn’t been an official update yet, but it’s a common request among users who want to bring the Copilot experience to all devices. 

Stay tuned for any announcements from GitHub regarding mobile support, as they continually work on enhancing accessibility across platforms.

**If this response was helpful, please mark it as the accepted answer, and feel free to follow me on GitHub for more updates and discussions on new features and developments!** 😊

**Reply by samus-aran** [reply]
Hi @SyedAliJunaidSherazi 👋, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by SyedAliJunaidSherazi** [reply]
@samus-aran Hi. Thanks for letting me know, actually it's my own content just being gramatically corrected  or some enhancements. I'll make sure to meet revised community guidelines and standards from now and will try to keep it 100% original. Really appreciated😊
